modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/components/Calculator/Calculator.soy

```diff
@@ -134,7 +134,7 @@
 
 	{let $item kind="html"}
 		{$label}
-		<span class="calculate-fieldname">Fieldname: {$title}</span>
+		<span class="calculate-fieldname">{msg desc=""}field-id{/msg}: {$title}</span>
```

Comment:
@brunofariasdeo, your fix seems good to me, but can you please give me more details about `{msg ...}`?

Thanks.